release-notes/8.0/releases.json

```diff
@@ -142,7 +142,7 @@
         "runtime-version": "8.0.10",
         "vs-version": "",
```

Comment:
I think vs-version lights up the included in field. don't we need that as well?

Reply:
done